### PR TITLE
[MAT-17] Add launcher implementation

### PR DESCRIPTION
**Pull request by gollariel**

### CODE REVIEW

**Comment by gollariel** [comment]
@thepeak99 This PR still WIP, but mostly I will work with filling launchers. All other things open to review :)

File: Dockerfile [hunk 1]
```diff
@@ -4,8 +4,8 @@ ADD . /go/src/github.com/patch-notes/mate
 WORKDIR /go/src/github.com/patch-notes/mate
 RUN apk update; apk add build-base upx git \
  && make build_mate \
- && upx build/mate -o /mate
+ && upx build/mate -o /mate_cli
```
Comment (thepeak99) [inline Dockerfile]: silly question, why did you rename it to `mate_cli`?

Comment (gollariel) [reply Dockerfile]: It's caused by the default configuration. By default, we want to have /mate directory, but we already have /mate file... This is the optional directory but will be better to go by convention way...

**Comment by gollariel** [comment]
@thepeak99 Updated!
